Revisions on solo staking guide (#785)
decred/dcrdocs: docs/advanced/solo-proof-of-stake-voting.md

    `wget https://github.com/decred/decred-release/releases/download/v1.3.0/{dcrinstall-linux-amd64-v1.3.0,manifest-dcrinstall-v1.3.0.txt,manifest-dcrinstall-v1.3.0.txt.asc}`

> 1. Verify the manifest. The output from this command should say "Good signature from Decred Release <release@decred.org>". Warnings about the key not being certified with a trusted signature can be ignore.

Reviewer: ```suggestion
1. Verify the manifest. The output from this command should say "Good signature from Decred Release <release@decred.org>". Warnings about the key not being certified with a trusted signature can be ignored.
```

1. Create a `bash` script called `decred.sh` which will start a `tmux` session for each application, start `dcrd`, start `dcrwallet`, prompt us for the password to unlock the wallet, and start `dcrctl`. 

    `echo "tmux new -d -s dcrd 'dcrd'; tmux new -d -s dcrwallet 'dcrwallet --promptpass'; tmux attach -t dcrwallet" > ~/decred.sh;`

1. Add the path to the Decred binaries to your `.profile`.

    `echo "PATH=~/decred:$PATH" >> ~/.profile && source ~/.profile`

1. Now make it executable with `chmod +x ~/decred.sh`

> Now you can start `dcrd`, `dcrwallet`, and unlock it just by running `./decred.sh`.

Reviewer: ```suggestion
Now you can start `dcrd`, `dcrwallet`, and unlock the wallet just by running `./decred.sh`.
```

Now from your cool wallet you can purchase tickets using the following command once your wallet is unlocked:

`dcrctl --wallet purchaseticket default 150 1 DsHotWalletAddressFromVotingWallet 10`

> Replacing `DsHotWalletAddressFromVotingWallet` with your hot wallet voting address which you will generate when you set up your remote servers for voting and using the command above will attempt to purchase `10` tickets with a max price of `150` DCR each and delegate voting rights to your hot wallets.

Reviewer: ```suggestion
Replacing `DsHotWalletAddressFromVotingWallet` with your hot wallet voting address, which you will generate when you set up your remote servers for voting, and using the command above will attempt to purchase `10` tickets with a max price of `150` DCR each and delegate voting rights to your hot wallets.
```

> If you wish to automate ticket purchases using ticketbuyer you will need to add the following info in your `~/.dcrwallet/dcrwallet.conf` where `DsHotWalletAddressFromVotingWallet` is once again the address you generated on your voting VPS.

Reviewer: ```suggestion
If you wish to automate ticket purchases using ticketbuyer you will need to add the following info in your `~/.dcrwallet/dcrwallet.conf`, where `DsHotWalletAddressFromVotingWallet` is once again the address you generated on your voting VPS.
```
